orc8r/gateway/python/magma/magmad/main.py

```diff
@@ -218,7 +218,7 @@ async def bootstrap_success_cb(certs_generated: bool):
     magmad_servicer = MagmadRpcServicer(
         service,
         services, service_manager, get_mconfig_manager(), command_executor,
-        service.loop,
+        service.loop, service.config.get('print_grpc_payload', False)
```

Comment:
Would be good to insert a new line after loop, for readability
(And if you feel like it, do you mind adding type annotation for loop?)